vite-project-kabinet/src/Content/LoadedList.jsx

```diff
@@ -1,22 +1,78 @@
 import "./paginated.style.css"
 import { CardPreview } from "./CardPreview.jsx"
 import { useState } from "react";
+import { useEffect } from "react";
 
-export const LoadedList = ({items, itemsPerPage}) => {
+export const LoadedList = ({itemsPerPage}) => {
+    const [items, setItems] = useState([])
+    const [totalRecords, setTotalRecords] = useState(0)
     const [visibleCount, setVisibleCount] = useState(itemsPerPage)
 
+    const getPostsWithCategoryName = async (perPage) => {
```

Comment:
Tohle je trochu matoucí název. Nevrací to `Post` protože to nevrací objekt s obsahem, ale jen s náhledem (perex), takže by to spíš mělo být `PostPreview`

Reply:
Táto funkcia vracia updated Posts, čiže upravené pole objektov. Preto názov tej funkcie začína getPosts ako množné číslo. A categoryName preto, lebo vlastnosť objektu category už nevracia číslo, ale reťazec.

Rada si nechám poradiť, že ako to premenovať lepšie, ale aspoň to množné čislo by som tam rada nechala :)

Reply:
Nešlo o to množné číslo, ale o to, že to nevrací celý článek ale jen jeho část. Takže název co jsem myslel by byl třeba `getPostsPreviewWithCategoryName`